### DISCUSSION TITLE
smartplug e firmware Tasmota

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#372] [Hardware & Firmware]
Ciao a tutti,
qualc'uno è riuscito a sostituire il firmware delle smartplug Tuya con il firmware Tasmota?
Ne ho comprata una della Elivco, con l'app funziona, ma non sono riuscito a usare il tool tuya converter.

L'ho anche resettata alle condizioni di fabbrica, ma nulla non riesco a sostituire il firmware.

Avete suggerimenti?

Giuseppe

**Comment by snakuzzo** [comment]
Sei sicuro che abbiano chip esp826x? Se non hanno questo chip non puoi caricarci tasmota. Aprile e vedi che chip hanno

**Reply by sky7176** [reply]
Ciao,
il dubbio mi è sorto. 
Solo che per aprirla devo romperla

**Reply by snakuzzo** [reply]
Vedi bene il codice del device e fai qualche ricerca
